models/articles.model.js

```diff
@@ -0,0 +1,12 @@
+const db = require('../db/connection');
+
+exports.selectArticleById = (article_id) => {
+  return db.query(`SELECT * FROM articles WHERE article_id = $1;`, [
+    article_id
```

Comment:
Love that you have made use of a parameterised query here to prevent SQL injection 🚀